src/App.tsx

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
-
+import React, { FC } from 'react';
 import './App.css';
+import {
+  HashRouter,
+  Route,
+  NavLink,
+} from 'react-router-dom';
+import { Home } from './components/Home';
+import { Tabs } from './components/Tabs';
+import { Tab } from './interfaces/TabInterface';
 
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+const tabs: Tab[] = [
+  { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
+  { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
+  { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
+];
 
-const App = () => (
+const App: FC = () => (
   <div className="App">
-    <h1>Tabs with router</h1>
+    <HashRouter>
+      <nav>
+        <ul>
+          <li>
+            <NavLink to="/" exact activeClassName="selected">Home</NavLink>
+          </li>
+          <li>
+            <NavLink to="/tabs" activeClassName="selected">Tabs</NavLink>
+          </li>
+        </ul>
+      </nav>
+
+      <Route path="/" exact component={Home} />
+      <Route path="/tabs" render={() => <Tabs tabs={tabs} />} />
```

Comment:
```suggestion
      <Route path="/tabs/:tabId" render={() => <Tabs tabs={tabs} />} />
```
ты можешь сделать так и брать параметры с URL и тогда тебе не прийдется создавать еще 3 роута внутри табс